src/browser_launchers/playwright_launcher.js

```diff
@@ -64,15 +64,40 @@ export class PlaywrightLauncher extends BrowserLauncher {
             launcher = BrowserLauncher.requireLauncherOrThrow('playwright', 'apify/actor-node-playwright-*').chromium,
         } = launchContext;
 
+        const { launchOptions = {}, ...rest } = launchContext;
+
         super({
-            ...launchContext,
+            ...rest,
+            launchOptions: {
+                ...launchOptions,
+                executablePath: getDefaultExecutablePath(launchOptions),
+            },
             launcher,
         });
 
         this.Plugin = PlaywrightPlugin;
     }
 }
 
+/**
+ *
+ * @param {Object<string, *>} launchOptions
+ * @returns {string|undefined} default path to browser.
+ * If actor-node-playwright-* image is used the APIFY_DEFAULT_BROWSER_PATH is considered as default.
+ * @ignore
+ */
+function getDefaultExecutablePath(launchOptions) {
+    const pathFromPlaywrightImage = process.env.APIFY_DEFAULT_BROWSER_PATH;
```

Comment:
I think we should inject this in the SDK, not here. `browser-pool` is apify-agnostic.

Reply:
I don't get this one. I am in the SDK, right? 🤷‍♂️ 😄

Reply:
I saw `playwright_launcher` and immediately thought it's `browser-pool` lol 🤣 🤦